man/loader.conf.xml

```diff
@@ -209,6 +209,47 @@
         <para>See <ulink url="https://systemd.io/RANDOM_SEEDS">Random Seeds</ulink> for further
         information.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>color</term>
+
+        <listitem><para>Foreground and background color separated by a <literal>/</literal>, defaults to
+        <literal>lightgray/black</literal>.</para>
```

Comment:
hmpf. do we really need this? this is a frickin boot loader, themable boot loaders are an illness, not something to strive for. 

So much code for a screen really nobody should see except when the system is fucked...

also i really don't like the non-normalization of this. i.e. the fg/bg thing. should be two distinct settings, like the myriad other color settings.

i'd be more open to this if you make this compile time settings. but runtime settings? jeez, no.

Reply:
Anyone dual booting with windows will need a boot loader of some sorts. And I figured there's a desire for it, so why not go for it. If you insist I can make it compile time option though.

Reply:
we make the "ok color" compiler time too. sounds like in the same territory.

Reply:
You mean the colors pid1 uses for it's status messages?

Reply:
yes

Reply:
so yeh pleae make this a compile time thing